code_examples/sdk_examples/src/core_features/public_credentials/01_create_credential.ts

```diff
@@ -1,10 +1,28 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 
 export function createNftCollectionCredential(
-  ctype: Kilt.ICType,
   assetDid: Kilt.AssetDidUri,
   artistDid: Kilt.DidUri
 ): Kilt.IPublicCredentialInput {
+  // CType definition.
+  const ctype = Kilt.CType.fromProperties(
+    `NFT Collection Certification CType`,
+    {
+      name: {
+        type: 'string'
+      },
+      pieces: {
+        type: 'integer'
+      },
+      creationDate: {
+        type: 'string'
+      },
+      artistIdentity: {
+        type: 'string'
+      }
+    }
+  )
```

Comment:
Move this out of the function. That makes it more readable.

Reply:
Done